examples/hello_flutter/lib/main.dart

```diff
@@ -7,7 +7,7 @@
 
 import 'dart:ui' as ui;
 
-void beginFrame(Duration timeStamp) {
+void _beginFrame(Duration timeStamp) {
```

Comment:
why this change?

Reply:
Whoops I didn't see that one sneak in. I think I had to do that because it raised an error that a public member was not properly documented. Consistently with my other change, I think in Fuchsia we should disable analysis on this package too.

Reply:
Updated the change to only affect build files.